v2.3.1 - edge-info - remove echo $NORM

JanneKiiskila opened the pull request:
It just seems to create a bit of mess now.

```
  - Lat/Long:			65.0124,25.4682
  - organization:		AS16086 DNA Oyj
\u001b[0m
```

Add also GitHub action, which blocks future occurences of such mistakes.

## Todos

- [x] Changelog updated
- [x] Run `shellcheck` or `pysh-check` before committing code - no more warnings than earlier (preferrably less)
- [x] Will tag a proper release, if need be.
- [ ] Will update required recipes/builds as well.
- [x] Will update also the versions to relevant places:
    - edge-info/edge-info has the version number (around line 37)
    - identity-tools/VERSION has the version number as well.

Ref: https://github.com/PelionIoT/meta-edge/pull/32

JanneKiiskila committed:
V 2.3.1 - edge-info - remove echo $NORM

It just seems to create a bit of mess now.

```
  - Lat/Long:			65.0124,25.4682
  - organization:		AS16086 DNA Oyj
\u001b[0m
```

petedyer11 commented on edge-info/edge-info:
```diff
@@ -368,7 +368,6 @@ firmware(){
     local bito
     bito=$(getconf LONG_BIT)
     kernelV=$(uname -r)
-    echo "${NORM}"
```
or change it to echo -e to  "enable interpretation of backslash escapes"

JanneKiiskila replied:
The best code line is the removed one.

petedyer11 reviewed (approved):
LGTM

JanneKiiskila committed:
GitHub action - edge-info

Fail if the output has any "\u" occurences, as that's a echo without
the -e doing it's nasty works.
